packages/web/src/store.ts

```diff
@@ -30,6 +36,28 @@ export const useData = create<State>((set) => ({
     }),
   connectedUsers: [],
   setConnectedUsers: (users) => set({ connectedUsers: users }),
+  directMessages: {},
+  addDirectMessage: (directMessage: DirectMessage) => {
+    const key = getPairKey(directMessage.from, directMessage.to);
+    set((state) => {
+      const newMessage: DirectMessage = {
+        id: Date.now(),
```

Comment:
```suggestion
        id: Date.now(),
```
use id from server, don't generate here